buildkite/origin_target_branch.sh

```diff
@@ -10,7 +10,12 @@ fi
 if [ $# -eq 2 ]; then
     origin_target_branch="$2"
 else
-    origin_target_branch="${BUILDKITE_PULL_REQUEST_BASE_BRANCH:-$BUILDKITE_BRANCH}"
+    if [[ "${BUILDKITE_PULL_REQUEST_REPO:-}" =~ ^git@github\.com:dlang/ ]]; then
```

Comment:
I hope this is the *source* repo of the PR. I.e., `kinke/druntime` when opening a druntime PR from my fork. https://buildkite.com/docs/pipelines/environment-variables#bk-env-vars-buildkite-pull-request-repo isn't too clear.